portal/src/test/scala/cromwell/pipeline/utils/auth/TestUserUtils.scala

```diff
@@ -2,24 +2,31 @@ package cromwell.pipeline.utils.auth
 
 import java.util.UUID
 
+import cromwell.pipeline.datastorage.dto.User.UserEmail
 import cromwell.pipeline.datastorage.dto.{ User, UserId }
 import cromwell.pipeline.utils.StringUtils
 
 object TestUserUtils {
   val userPassword = "-Pa$$w0rd-"
 
-  def getDummyUser(password: String = userPassword, passwordSalt: String = "salt", active: Boolean = true): User = {
-    val uuid = UUID.randomUUID().toString
+  def getDummyUser(
+    uuid: String = UUID.randomUUID().toString,
+    emailOpt: UserEmail = "JohnDoe-@cromwell.com",
+    password: String = userPassword,
+    passwordSalt: String = "salt",
+    firstName: String = "FirstName",
+    active: Boolean = true
+  ): User = {
     val passwordHash = StringUtils.calculatePasswordHash(password, passwordSalt)
     User(
-      userId = UserId(uuid),
-      email = s"JohnDoe-$uuid@cromwell.com",
-      passwordHash = passwordHash,
-      passwordSalt = passwordSalt,
-      firstName = "FirstName",
-      lastName = "LastName",
-      profilePicture = None,
-      active = active
+      UserId(uuid),
+      s"JohnDoe-$uuid@cromwell.com",
+      passwordHash,
+      passwordSalt,
+      "FirstName",
+      "LastName",
+      None,
+      active
```

Comment:
Include other `lastName` as a parameter too and don't forget to use it in a `User` constructor. Also, rename `emailOpt` to `email`.

Reply:
done: 
a) added parameters to constructor, 
b) used this params in constructor in getDummyUser(), 
c)renamed emalopt